setup.py

```diff
@@ -1,14 +1,25 @@
+"""
+    The setup file.
+"""
+import os
 import setuptools
 
-setuptools.setup(
-    install_requires=[
-        "stable-diffusion-sdkit", # wrapper around stable-diffusion, to allow pip install
+install_requires = []
+
+
+if not os.getenv('SKIP_EXTERNAL_DEPS'):
+    install_requires: list[str] = [
```

Comment:
The data-type declaration here is redundant. It's clear that it's a list of strings.